chirp/drivers/kg935g.py

```diff
@@ -82,14 +83,17 @@
 VOX_LIST = ["Off"] + ["%s" % x for x in range(1, 10)]
 BANDWIDTH_LIST = ["Narrow", "Wide"]
 VOICE_LIST = ["Off", "On"]
-LANGUAGE_LIST = ["Chinese", "English"]
+LANGUAGE_LIST = ["CN", "English", "TC"]
 SCANMODE_LIST = ["TO", "CO", "SE"]
 PFKEYLONG_LIST = ["undef", "FRQ2-PTT", "Selec Call", "Scan", "Flashlight",
                   "Alarm", "SOS", "FM Radio", "Moni", "Strobe", "Weather",
                   "Tlk A", "Reverse", "CTC Scan", "DCS Scan", "BRT"]
 PFKEYSHORT_LIST = ["undef", "Scan", "Flashlight", "Alarm", "SOS", "FM Radio",
                    "Moni", "Strobe", "Weather", "Tlk A", "Reverse",
                    "CTC Scan", "DCS Scan", "BRT"]
+PFKEYLONG_LIST_UV8H = ["Selec Call", "Vice-Fre Tx"]
```

Comment:
I'm guessing these are direct from the existing software or radio interface? "Vice-Fre Tx" sounds like a "we google-translated the manual" error, and I don't think we should reproduce such things in CHIRP. People like using chirp over the OEM chinese software *because* we use words that make sense, IMHO :)

Reply:
yep -  again radio display  vs English... English will win.

Reply:
Thanks :)

Reply:
I updated and committed.  I squashed the 2 commits.. I hope I did that correctly.